toolchains/toolchains.bzl

```diff
@@ -5,6 +5,15 @@ load(":prebuilt_toolchains.bzl", _prebuilt_toolchains = "prebuilt_toolchains")
 # Re-expose the prebuilt toolchains macro
 prebuilt_toolchains = _prebuilt_toolchains
 
+# buildifier: disable=unnamed-macro
+def built_toolchains():
```

Comment:
Can you also make a `built_toolchains.bzl` file that has a similar structure to `prebuilt_toolchains.bzl` and defines all the source locations for the various build tools? 🙏